apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.command.ts

```diff
@@ -15,4 +16,9 @@ export class MarkManyNotificationsAsCommand extends EnvironmentWithSubscriber {
   @IsOptional()
   @IsBoolean()
   readonly archived?: boolean;
+
+  @IsOptional()
+  @Type(() => Date)
```

Comment:
just out of curiosity why do we need ` @Type(() => Date)`  the decorator `@IsDate()` wont do the validation properly by itself?